app/src/main/java/com/github/se/orator/ui/network/ChatGPTService.kt

```diff
@@ -38,6 +38,11 @@ data class ChatResponse(
     val usage: Usage
 )
 
+enum class SessionType {
+  PRACTICE,
+  BATTLE
```

Comment:
This Enum class if not being used. I think that it will be useful when implementing the feedback from GPT in the next steps you mentioned. If not, it should be removed.